arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -292,54 +388,138 @@
       "type": "object",
       "properties": {
         "storageAccount": {
-          "$ref": "#/definitions/StorageAccountProperties"
+          "$ref": "#/definitions/StorageAccountBaseProperties"
         },
         "loginServer": {
           "type": "string"
         },
         "username": {
           "type": "string"
         },
-        "key": {
-          "type": "string"
-        },
         "creationDate": {
           "format": "date-time",
           "type": "string"
         }
       }
     },
-    "StorageAccountProperties": {
+    "StorageAccountBaseProperties": {
+      "type": "object",
+      "properties": {
+       "name": {
+          "type": "string"
+        }
+      }
+    },
+    "RegistryCreateParameters": {
       "type": "object",
       "properties": {
+        "id": {

```

Comment:
I think "id" and "type" should be removed from create parameters since the client is not able to set them anyway.